scanner/src/main/kotlin/scanners/scancode/ScanCodeResultParser.kt

```diff
@@ -162,6 +162,11 @@ private fun generateScannerOptions(options: JsonNode?): String {
     }.orEmpty()
 }
 
+private fun getInputPath(result: JsonNode): String {
+    val header = result["headers"]?.singleOrNull()
+    return header?.get("options")?.get("input")?.singleOrNull()?.textValue()?.let { "$it/" }.orEmpty()
```

Comment:
Can `it` be an empty string?

Reply:
Not that I'm aware of. Also please note that this code was just moved to here.